src/abstractdataframe/reshape.jl

```diff
@@ -100,17 +103,15 @@ function _stackview(df::AbstractDataFrame, measure_vars::AbstractVector{Int},
     cnames = _names(df)[id_vars]
     push!(cnames, variable_name)
     push!(cnames, value_name)
-    if variable_eltype <: CategoricalValue{String}
-        nms = names(df, measure_vars)
-        catnms = categorical(nms)
-        levels!(catnms, nms)
-    elseif variable_eltype <: Symbol
-        catnms = _names(df)[measure_vars]
-    elseif variable_eltype <: String
-        catnms = names(df, measure_vars)
+    if variable_eltype === Symbol
+        catnms = PooledArray(_names(df)[measure_vars])
```

Comment:
there is probably not much benefit from it here but we can have it for consistency.

Reply:
Good catch. Yeah that's probably counter-productive here so I've reverted the change. What could make sense is to define `copy(::RepeatedVector)` to return a `PooledArray`, but that's non-breaking so we can do that later.